client/mapreduce/src/main/java/org/apache/accumulo/core/client/mapred/AccumuloOutputFormat.java

```diff
@@ -133,8 +144,8 @@ public static void setConnectorInfo(JobConf job, String principal, Authenticatio
     if (token instanceof KerberosToken) {
       log.info("Received KerberosToken, attempting to fetch DelegationToken");
       try {
-        Instance instance = getInstance(job);
-        Connector conn = instance.getConnector(principal, token);
+        Connector conn = Connector.builder().usingClientInfo(getClientInfo(job))
+            .usingToken(principal, token).build();
```

Comment:
I don't think this behavior is needed. There's no reason to expect the keytab *won't* be on the worker nodes. If the user wants to use a delegation token, they can create one and pass it in. This auto-allocation of delegation tokens without user explicit acknowledgement is probably a bit dangerous anyway and may not be what users intend.

If `getClientInfo` was added solely to support this feature, it is probably not needed.